health/micro-ui/web/micro-ui-internals/packages/modules/health-hrms/src/components/pageComponents/SelectableList.js

```
@@ -0,0 +1,37 @@
+        <div
+          key={user.id}
+          onClick={() => handleSelect(user)}
+          style={{
+            cursor: "pointer",
+            padding: "10px",
+            marginBottom: "5px",
+            border: "1px solid #ccc",
+            borderRadius: "5px",
+            backgroundColor: user.id === selectedUserId?.id ? "lightgrey" : "white",
+          }}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)